docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2034,6 +2091,91 @@ When `LOGGING` is enabled, attestation and block performance is reported as log
 `METRICS` is enabled, attestation and block performance is reported using [metrics] in the
 [`VALIDATOR_PERFORMANCE`](#metrics-categories) metrics category.
 
+### validators-proposer-config
```

Comment:
I think we should provide information about the json format.
the format is reported here:
https://github.com/ConsenSys/teku/pull/4894